tests/integration/test_trimmed_geometry.py

```diff
@@ -0,0 +1,246 @@
+from ansys.geometry.core.shapes import Circle, Line
+from ansys.geometry.core.shapes.box_uv import LocationUV
+from ansys.geometry.core.shapes.curves.trimmed_curve import ReversedTrimmedCurve, TrimmedCurve
+from ansys.geometry.core.shapes.parameterization import Interval
+from ansys.geometry.core.shapes.surfaces.trimmed_surface import (
+    ReversedTrimmedSurface,
+    TrimmedSurface,
+)
+from ansys.geometry.core.sketch.sketch import Sketch
+
+"""A Helper function to create a sktech line given two points and a ``Design``."""
+
+
+def create_sketch_line(design: Design, p1: Point3D, p2: Point3D):
+    point1 = point3d_to_grpc_point(p1)
+    point2 = point3d_to_grpc_point(p2)
+    design._commands_stub.CreateSketchLine(CreateSketchLineRequest(point1=point1, point2=point2))
+
+
+"""A Helper function that creates the Hedgehog model."""
+
+
+def create_hedgehog(modeler: Modeler):
+    design = modeler.create_design("Hedgehog")
+    sketch = Sketch().arc_from_three_points(
+        Point2D([0.01, 0.01]), Point2D([0, -0.005]), Point2D([-0.01, 0.01])
+    )
+    sketch.arc_from_three_points(
+        Point2D([0.01, 0.01]), Point2D([0, 0]), Point2D([-0.01, 0.01])
+    ).circle(Point2D([0, 0]), 0.02)
+    body = design.extrude_sketch("Cylinder", sketch, 0.02)
+    # Add points to edges and corners - no missing corners
+    points_between_corners = 4
+    for edge in body.edges:
+        total_point_gaps = np.floor(edge.length.m * 1000 / points_between_corners)
+        current_gap = 0
+        while current_gap < total_point_gaps:
+            design.add_design_point(
+                f"Point_{current_gap}",
+                edge.shape.evaluate_proportion(current_gap / total_point_gaps).position,
+            )
+            current_gap += 1
+    # Add hedgehog hairs (face-normals) to these points
+    for face in body.faces:
+        for edge in face.edges:
+            total_point_gaps = np.floor(edge.length.m * 1000 / points_between_corners)
+            current_gap = 0
+            while current_gap < total_point_gaps + 1:
+                p1 = edge.shape.evaluate_proportion(current_gap / total_point_gaps).position
+                uv = face.shape.project_point(p1).parameter
+                u, v = face.shape.get_proportional_parameters(uv)
+                normal = face.normal(u, v)
+                p2 = design.add_design_point("hair", p1 + normal / 800).value
+                create_sketch_line(design, p1, p2)
+                current_gap += 1
+    # Add isoparametric curves, not on linux
+    if modeler.client.backend_type != BackendType.LINUX_SERVICE:
+        param = 0.20
+        while param <= 1:
+            for face in body.faces:
+                face.create_isoparametric_curves(True, param)  # u
+                face.create_isoparametric_curves(False, param)  # v
+            param += 0.20
+    return design
+
+
+"""A fixture of the hedgehog design to test the surface and curve properties individually."""
```

Comment:
Earlier occurrences of "Hedgehog" were capitalized and now this one isn't. If it is a proper noun, capitalize the initial letter. Otherwise, don't capitalize it. If you don't know, do a Google search and see how what the most often used format is.